backend/src/services/LandlordService.ts

```diff
@@ -1,17 +1,38 @@
 // Here we would have a function like createUser that would create a new user with data taken from the controller,
 // then create a new model (taken from model folder), save the user and return it.
-import { HydratedDocument } from 'mongoose';
+import { HydratedDocument, Types } from 'mongoose';
 import LandlordModel, { ILandlord } from '../model/Landlord';
 
 export const createLandlord = (firstName: string, lastName: string, organization?: string): Promise<ILandlord> => {
   const landlord: HydratedDocument<ILandlord> = new LandlordModel({ firstName, lastName, organization });
   return landlord.save();
 };
 
+export const createNewLandlord = (
+  firstName: string,
+  lastName: string,
+  organization?: string
+): Promise<ILandlord> => {
+  return new Promise((resolve, reject) => {
+    LandlordModel.create({ firstName, lastName, organization })
+      .then((result) => {
+        resolve(result);
+      })
+      .catch((error: Error) => {
+        console.log(error);
+        reject(new Error(`Failed to create landlord: ${error}`));
+      });
+  });
+};
```

Comment:
Cant find where this function gets used

Reply:
I made some errors reverting commits, it should have been removed by now.